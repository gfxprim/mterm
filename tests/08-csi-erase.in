10 10
test\ntest\ntest\e[2J
