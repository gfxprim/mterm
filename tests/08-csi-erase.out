 ----------
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
 ----------
size 10x10 cursor 2x4
